typescript/packages/subsurface-viewer/src/storybook/sharedSettings.tsx

```diff
@@ -94,6 +94,7 @@ export const volveWellsFromResourcesLayer = {
     id: "volve-wells",
     data: "@@#resources.wellsData",
     ZIncreasingDownwards: false,
+    wellNameVisible: true,
```

Comment:
Maybe we shouldn't enable well names across the board. It can make it more difficult to isolate problems in the future.